generator/holes.cpp

```diff
@@ -0,0 +1,75 @@
+#include "generator/holes.hpp"
+
+#include "generator/intermediate_data.hpp"
+#include "generator/osm_element.hpp"
+
+namespace generator
+{
+// HolesAccumulator
+HolesAccumulator::HolesAccumulator(cache::IntermediateDataReader & holder) :
+  m_merger(holder)
+{
+}
+
+FeatureBuilder1::Geometry & HolesAccumulator::GetHoles()
+{
+  ASSERT(m_holes.empty(), ("Can call only once"));
+  m_merger.ForEachArea(false, [this](FeatureBuilder1::PointSeq const & v,
+                       std::vector<uint64_t> const &)
```

Comment:
Please add a short in line comment for the second param: `std::vector<uint64_t> const & /* some comment */`

Reply:
++